dissect/util/compression/__init__.py

```diff
@@ -0,0 +1,29 @@
+from dissect.util.compression import lz4 as lz4_python
+from dissect.util.compression import lzo as lzo_python
+
+try:
+    import lz4.block as lz4
+    import lz4.block as lz4_native
+except ImportError:
+    lz4 = lz4_python
+    lz4_native = None
+
+try:
+    import lzo
+    import lzo as lzo_native
+except ImportError:
+    lzo = lzo_python
+    lzo_native = None
+
```

Comment:
Can you add some comments explaining this? Just so it's clear to anyone looking around here what's happening.